src/vampy/__init__.py

```diff
@@ -8,21 +8,27 @@
 from .automatedPostprocessing import postprocessing_common
 from .automatedPostprocessing import visualize_probes
 # Imports from pre-processing
-from .automatedPreprocessing import DisplayData
-from .automatedPreprocessing import ImportData
-from .automatedPreprocessing import NetworkBoundaryConditions
-from .automatedPreprocessing import ToolRepairSTL
-from .automatedPreprocessing import automated_preprocessing
-from .automatedPreprocessing import preprocessing_common
-from .automatedPreprocessing import simulate
-from .automatedPreprocessing import visualize
-from .automatedPreprocessing import vmtk_pointselector
+try :
+    from .automatedPreprocessing import DisplayData
+    from .automatedPreprocessing import ImportData
+    from .automatedPreprocessing import NetworkBoundaryConditions
+    from .automatedPreprocessing import ToolRepairSTL
+    from .automatedPreprocessing import automated_preprocessing
+    from .automatedPreprocessing import preprocessing_common
+    from .automatedPreprocessing import simulate
+    from .automatedPreprocessing import visualize
+    from .automatedPreprocessing import vmtk_pointselector
+except ModuleNotFoundError :
+    print("morphman is not installed, automated preprocessing is not available")
```

Comment:
Should this be `morphMan` instead of `morphman`?

Maybe it should be more clear that this is just a warning? Perhaps like this:
``` Python
print("WARNING: morphMann is not installed, automated preprocessing is not available")
```